Todo.js

```diff
@@ -0,0 +1,77 @@
+export default class Todo {
+  constructor() {
+    this.todos = new Map();
+    this.idCounter = 0;
+  }
+
+  addItem = (title, priority) => {
+    const givenId = this.idCounter;
+    this.todos.set(
+      givenId,
+      {
+        title,
+        priority
+      }
+    );
+    this.idCounter++;
+    return givenId;
+  }
+
+  removeItem = (id) => {
+    if (this.todos.has(id)) {
+      this.todos.delete(id);
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  getItem = (id) => {
+    this.todos.has(id)
+      ? { id, ...this.todos.get(id) }
+      : null
+  }
+
+
+  next = () => {
+    if (this.todos.size === 0) {
+      throw new Error('The todo list is empty!');
+    } else {
+      const todosEntries = Object.entries(this.todos);
+      let maxPriority = (todosEntries[0][1]).priority;
```

Comment:
do not need to use variables in closure with reduce
unneeded parentesis `todosEntries[0][1].priority;`